plugin-packs/postcss-preset-env/README.md

```diff
@@ -9,9 +9,17 @@
 can understand, determining the polyfills you need based on your targeted
 browsers or runtime environments.
 
-```bash
-npm install postcss-preset-env
-```
+## How does it work?
+
+[PostCSS Preset Env] is a Plugin Pack for [PostCSS]. It leverages the list of the features we keep an eye from [CSSDB][cssdb] and applies plugins, so you can use those new features without having to worry about browser support.
+
+CSSDB exposes the browser support that each feature has which can come from [Can I Use](https://caniuse.com/css-all) or from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/all) (through [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)).
+
+However, if your site doesn't need to support older browsers you wouldn't want for certain features to be expanded, effectively increasing your CSS bundle without any real benefit. 
```

Comment:
Maybe we can omit this here and save it for a more dedicated doc or entry focused on performance? I don't disagree with the statement but feel it needs more context.

The one clear value proposition that any project like this has is making it cheaper and simpler for developers to target a wider range of browsers and in doing so making the web more open and accessible.

The few kb saved (after gzip) are often negligible even when comparing something like IE11 and latest Chrome.

I also want to avoid people setting targets that are too modern for performance gains that are maybe too small to measure.

Reply:
The reasoning for this sort of section is to explain why does it need a browser list or why would you want to be accurate with it. I don't think right now was clear what was going on. I'm keen on rewording it though! Let's connect on this.

Reply:
So true!
Maybe we can focus on how it is related to `stage` without mentioning bundle size?

(`stage` enables features and `browsers` disables them.)

So maybe something like this :

By providing a list of browser targets for your project, plugins that aren't needed will be skipped. Over time your targets will change and by updating the settings your CSS bundle will only ever contain the needed fallbacks.